src/importNameRule.ts

```diff
@@ -205,6 +223,19 @@ function walk(ctx: Lint.WalkContext<Option>) {
         return false;
     }
 
+    function transformName(input: string) {
+        switch (option.config.case) {
+            case StringCase.camel:
+                return makeCamelCase(input);
+            case StringCase.pascal:
+                return makePascalCase(input);
+            case StringCase.any:
+                return `${makeCamelCase(input)} or ${makePascalCase(input)}`;
```

Comment:
Output of `transformName` will be used as replacement in fix and this string will produce incorrect TS code.